spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
> However, I don't really know exactly what it means to say that it "is a Boolean object". As far as I can tell, what makes something a Boolean object is that it has the [[BooleanValue]] internal slot, so the sentence would seem a little redundant.

In that case you could also write:
"The Boolean prototype object is itself a Boolean object; its [[BooleanData]] internal slot has the value false."


> However, I don't understand the concern about implementations having the right to redefine internal methods for Boolean.prototype just because it's an exotic object. Do implementations have this right on, say, Arrays?

Exotic objects have a different definition for one (or more) of the internal methods (cf. [4.3.7](http://tc39.github.io/ecma262/#sec-exotic-object)). The term "Array exotic object" has a specific meaning, it refers to the definition in [9.4.2](http://tc39.github.io/ecma262/#sec-array-exotic-objects) which only specifies to use a different implementation for [[DefineOwnProperty]].